example-library/src/main/resources/application-example.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+eureka:
```

Comment:
same concerns as stated in the services repo PR about embedding eureka into the example yaml

Reply:
This has been removed.

Reply:
This has been removed from the yaml